src/features/projects/components/projectDetails/TopProjectReports.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styles from './../../styles/ProjectDetails.module.scss';
+import VerifiedIcon from '@mui/icons-material/Verified';
+import { getPDFFile } from '../../../../utils/getImageURL';
+import parse from 'date-fns/parse';
+import format from 'date-fns/format';
+import { Trans } from 'react-i18next';
+import { useTranslation } from 'next-i18next';
+
+interface Props {
+  data: object[];
```

Comment:
Define the type here. It can be replaced later.

Reply:
@prachigarg19 
Made a minor change to destructure the props so that we don't use the syntax `projectReviews?.projectReviews?.map((review)`

For future reference, you should either destructure the props, or just leave them named as `props` within the parameters for the functional component. In this way, you would use either `props.projectReviews` or simply `projectReviews` making the code easier to read.